src/components/SettingsOverlay.module.css

```
@@ -190,7 +190,7 @@
   box-shadow: var(--focus-ring-glow);
 }
 
-.overlay-help-trigger .icon {
+.overlay-help-trigger :global(.icon) {
```

Stylelint (17.7.0)

[error] Unknown pseudo-class selector ":global" (selector-pseudo-class-no-unknown)

(selector-pseudo-class-no-unknown)